libs/arangodb/langchain_arangodb/chains/graph_qa/arangodb.py

```diff
@@ -560,10 +592,21 @@ def _call(
                 "user_input": user_input,
                 "aql_query": aql_query,
                 "aql_result": aql_result,
+                "chat_history": chat_history,
             },
             callbacks=callbacks,
         )
 
+        # Add summary
+        text = "Summary:" if self.execute_aql_query else "AQL Explain:"
```

Comment:
Since we already have `AQL Explain` printed above, we can just go directly to printing `Summary:`

Good catch on adding this, I think it makes the UX much better!

```suggestion
        text = "Summary:"
```